pkg/webui/oauth/views/login/index.js

```diff
@@ -60,10 +60,14 @@ const appRoot = selectApplicationRootPath()
 @bind
 export default class OAuth extends React.PureComponent {
   static propTypes = {
-    location: PropTypes.string.isRequired,
+    location: PropTypes.location,
```

Comment:
is should be required. see `L95`, `L102`

Reply:
I updated it, the reason why I put it as not required because sometimes `location.state` was undefined so wanted to define a default prop for it. This works fine as well.